frontend/README.md

```diff
@@ -1,33 +1,56 @@
 # Overview
 
-The frontend is written in [vue3](https://vuejs.org) composition api in typescript. Using the [BC Parks Bootstrap theme](https://digitalspace.github.io/bcparks-bootstrap-theme/)
+The frontend is written in [vue3](https://vuejs.org) composition api in typescript, building with [Vite](https://vitejs.dev). Using the [BC Parks Bootstrap theme](https://digitalspace.github.io/bcparks-bootstrap-theme/). The theme still need to be discussed with UX designers.
 
 ## **Setup local development**
 
-```
 - Create a .env file inside this frontend folder with the following options:
 
-VITE_BACKEND_URL=http://localhost:3000
-VITE_KEYCLOAK_URL=[keycloak authentication url for dev server]
-VITE_KEYCLOAK_CLIENT_ID=[keycloak client name]
-VITE_KEYCLOAK_REALM=[keycloak realm name]
+  ```
+  VITE_BACKEND_URL=http://localhost:3000
+
+  // to enable the keycloak authentication, add the following:
+  VITE_KEYCLOAK_URL=https://dev.loginproxy.gov.bc.ca/auth
+  VITE_KEYCLOAK_CLIENT_ID=[keycloak client name]
+
+  // to disable the keycloak authentication, add this:
+  VITE_NODE_ENV=openshift-dev
+  ```
 
 - Install all requirement packages: `npm install`
 - Start the application: `npm start`
-```
+- Run test: `npm test`
+
+## **Setup visual studio code**
```

Comment:
Did you know: You can use (Development Containers)[https://microsoft.github.io/code-with-engineering-playbook/developer-experience/devcontainers/] to streamline environments on VSCode? We can discuss it more on #241

Reply:
will work on this, merge this pull request first